ui/components/failed.jsx

```diff
@@ -54,31 +95,22 @@ class FailedTest extends React.Component {
   }
 
   _renderSelectedView(paths) {
-    const Component =
-      _.find(FailedTest.VIEW_COMPONENTS, {name: this.state.view}).component;
 
     return <div>
-      <Jumbotron> <Component paths={paths} /> </Jumbotron>
+      <Jumbotron> {this.loadChild(this.state.view, paths)} </Jumbotron>
```

Comment:
No me gusta. You're duplicating the setup between the children **and** you're moving the strategy details from outside to inside the component.